src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -118,6 +118,7 @@ def run_python_test(test_target, pytest, python_setup, source_root_config, subpr
     env=pex_exe_env,
     input_files=merged_input_files,
     description=f'Run Pytest for {test_target.address.reference()}',
+    timeout_seconds=test_target.timeout if hasattr(test_target, 'timeout') else 60
```

Comment:
Clarity is truly in the eye of the beholder, but consider `getattr(test_target, 'timeout', 60)`.

Reply:
That is much better! I didn't know that exists.